public/endrit.html

```diff
@@ -15,5 +16,102 @@
+            <p>Splinter knew they would never be accepted in society, so he took them to the sewers, made a home down there, and made the four turtles his children. Knowing the world would be dangerous and there would be a time where they want to explore the outside world, Splinter trained the boys in the art of Ninjitsu. Mikey and his brothers were named after great Renaissance artists from a book Splinter fished out of the storm drain.</p>
+
+            <br>
+            <h2>Appearance</h2>
+            <p>Mikey is a light green Mutant ninja turtle. Unlike all of his previous incarnations, Mikey has freckles. Like all in incarnations, however, he wears an orange mask, though like his 1987 and 2003 incarnations, the tails of his mask are shorter. Mikey is shorter than his brothers and is considerably shorter than Donatello. Mikey also seems to be a little more pudgy than his brothers. The animators gave Mikey baby blue eyes to make him look slightly more 'innocent'. However, like all of the Turtles when they're in combat, the pupils and irises vanish from his eyes, leaving him with plain white bits, which are similar to that of the 2003 incarnations.</p>
+
+            <br>
+            <h2>Personality</h2>
+            <p>Mikey is the youngest of all the turtles, making him the least mature, however, it was confirmed that his maturity slowly increases as the episodes go on. He is labeled as "The Wild One" and "The Party Dude" as he is the most "fun" out of all the turtles. He is also very social, and wants the outside world (humans) to accept him and his brothers. He is also impulsive, because of a severe case of ADD. He is the first of the turtles to taste pizza and is very proud of his talent for naming things. In battle his weapon of choice is a pair of nunchakus, and a kusarigama. He might seem unintelligent to his apparent disregard for thinking things through, but he is actually intelligent. Unfortunately he also isn't always the most considerate of the ninja turtles like when he tried to use Raph's fear of Cockroach Terminator to get Raph to let him read his comics, or when he pranks his brothers. But he also has an honest heart and is more willing than the other turtles to accept outsiders like Karai/Miwa and Leatherhead.</p>
+
+            <br>
+            <h2>Bio</h2>
+            <p>Impulsive, creative and very social, Michelangelo is the heart and soul of the Ninja Turtles. The runt of the litter, he may be the smallest of his mutant brothers, but that doesn't make him the weakest (just harder to catch). Enemies beware! Michelangelo is a master of the Kusarigama, nunchucks with a secret and sharp surprise!</p>
+
+            <br>
+            <h2></h2>
+            <p></p>
+            <br>
+        </div>
+        <div class="right">
+            <img src="./img/michelangelo.jpeg">
+            <table>
+                <tbody>
+                    <tr><th>Michelangelo</th></tr>
```

Comment:
Provo me bo me colspan
```suggestion
                   <th colspan="2">Michelangelo</th>
```